setup.py

```diff
@@ -323,15 +323,29 @@ def get_long_description():
             "PySide2;python_version<'3.9'",
             "traitsui;python_version<'3.9'",
         ],
+        "examples": [
+            # dependencies for examples
+            "numpy",
+            "pillow",
+        ]
     },
     ext_modules=[setuptools.Extension("traits.ctraits", ["traits/ctraits.c"])],
     package_data={
+        "traits": [
+            "examples/introduction/*",
```

Comment:
i'm not sure why there is an `traits/examples/introduction/__init__.py` file.

Reply:
Currently it contains some text. That's how the demo application used to load the main description of a "folder".
They can be moved to an `index.rst` and then the `__init__.py` can be removed.
That should be a separate PR though.